### DISCUSSION TITLE
Fixed time operations with limits?

### DISCUSSION THREAD

**Discussion by fatLittleDog** [frePPLe/frepple#383] [General]
Hi!  I am trying to model an operation that I think should be a fixed time but needs to have an upper limit - let me explain:
Drawing from the Frepple manual, if I have a truck that needs to transport packages, it doesn't matter if it needs to transport 1 package or 100 packages, the trip will take 30 minutes.  So this is a fixed time operation.

But what if my truck can only hold 50 packages on each trip?  If I have 100 packages, and I have 1 truck, it will need to make 2 trips.  If I have 2 trucks, each truck can make 1 trip.

Similarly, let's say I am a bakery and I have 3 mixers and each mixer can hold 200kg of dough.  It doesn't matter if I am mixing 20kg or 200kg, each mix takes 30 minutes.  But I can only mix 200kg at a time in each mixer.  So if I have 800kg to mix, I need to do 4 200kg batches, or use 2 mixers to do one mix and 1 mixer to do two mixes.

What would be the correct way to model this kind of resource in Frepple?

Thank you!

Peter

**Comment by jdetaeye** [comment]
>>But what if my truck can only hold 50 packages on each trip? If I have 100 packages, and I have 1 truck, it will need to make 2 trips. If I have 2 trucks, each truck can make 1 trip.

See this example at https://frepple.org/docs/current/user-guide/examples/supplier/supplier-capacity.php
This is with a bucketized resource.

>>Similarly, let's say I am a bakery and I have 3 mixers and each mixer can hold 200kg of dough. It doesn't matter if I am mixing 20kg or 200kg, each mix takes 30 minutes. But I can only mix 200kg at a time in each mixer. So if I have 800kg to mix, I need to do 4 200kg batches, or use 2 mixers to do one mix and 1 mixer to do two mixes.

You can set a operation "max-size" to break big requirements into smaller 200kg batches.
